var/spack/repos/builtin/packages/libgcrypt/package.py

```diff
@@ -32,8 +32,11 @@ class Libgcrypt(AutotoolsPackage):
        key algorithms, large integer functions, random numbers and a lot
        of supporting functions. """
     homepage = "http://www.gnu.org/software/libgcrypt/"
-    url      = "http://gd.tuwien.ac.at/pub/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    url 	 = "http://gnupg.org/ftp/gcrypt/libgcrypt/libgcrypt-1.7.6.tar.bz2"
+    list_url = "https://ftp.heanet.ie/disk1/gnupg/libgcrypt/libgcrypt-1.6.2.tar.bz2"
+    list_depth = 2
```

Comment:
So, the purpose of `list_url` is to allow Spack to find additional versions of a package when the directory from `url` is not listable. Here, the directory is listable. The only time this `list_url` could possibly be used is if the main `url` goes down. In that way, `list_url` is being used as a pseudo-mirror. I guess there's no reason not to, but it feels kind of hacky. We've talked about adding support for mirrors before, but nothing has ever happened, so in the meantime I guess this is fine.